packages/client/tests/functional/decimal/precision/tests.ts

```diff
@@ -63,5 +63,9 @@ testMatrix.setupTestSuite(
         mongo - decimals are not supported
     `,
     },
+    skipProviderFlavor: {
+      from: ['js_neon', 'js_pg'],
+      reason: "Unsure how this test works, it's likely a loss of precision",
```

Comment:
Looks like something we need to investigate

Reply:
This is something Miguel has just fixed on the Rust side.

Reply:
Actually, I've just verified that even with https://github.com/prisma/prisma-engines/pull/4285, this test still fails.

Reply:
Added a task to review this test in https://github.com/prisma/team-orm/issues/381 and merged the fix for floats, given it made some of engine tests that before were failing to pass.